### PR TITLE
feat: add service for setting announce volume

### PR DESCRIPTION
**Pull request by eyr1n**
## Description(required)

feat: add service for setting announce volume

## Review Procedure(required)

- check if the volume changes with the following command(please use tier4_hmi_msgs in feat/announce_volume_srv branch)

```bash
ros2 service call /signage/volume tier4_hmi_msgs/srv/Volume "{volume: 0.5}"
```

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Read [commit-guidelines][commit-guidelines]
- [x] Assign PR to reviewer

If you are adding new package following items are required:

- [ ] Documentation with description of the package is available
- [ ] A sample launch file and parameter file are available if the package contains executable nodes

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write release notes

[commit-guidelines]: https://www.conventionalcommits.org/en/v1.0.0/
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: src/signage/src/signage/announce_controller.py [hunk 1]
```diff
@@ -100,3 +110,20 @@ def announce_going_to_depart_and_arrive(self, message):
...
             self._prev_depart_and_arrive_type = message
+
+    def get_volume(self, _, response):
+        try:
+            volume = self._sink.volume.value_flat
+            response.status.code = ResponseStatus.SUCCESS
+            response.volume = volume
+        except Exception:
+            response.status.code = ResponseStatus.ERROR
+        return response
```
[the remark this answers is not in the document]
Comment (eyr1n) [reply src/signage/src/signage/announce_controller.py:121]: Should I make custom msg for the topic? I'm using std_msgs/msg/Float32 to publish the volume.

[the remark this answers is not in the document]
Comment (tkhmy) [reply src/signage/src/signage/announce_controller.py:121]: I feel that it is not needed to have a custom msg. @Hidenaga41 can mot handle the standard message?

[the remark this answers is not in the document]
Comment (Hidenaga41) [reply src/signage/src/signage/announce_controller.py]: Yes. MOT can handle the standard messages.

**Comment by tkhmy** [comment]
@eyr1n Do you store the sound volume after the sound is setted?
Because each time the ecu restart, we will need to set the sound volume again

**Comment by eyr1n** [comment]
I will fix to store the volume in the node.
But how do I reset the volume when the ecu is restarted?

**Comment by tkhmy** [comment]
> @tkhmy I will fix to store the volume in the node. But how do I reset the volume when the ecu is restarted?

@eyr1n you can just call the set volume function in the init function

**Review by tkhmy — APPROVED** [review]
LGTM

**Pull request merged**